assets/scripts/app/focus.js

```diff
@@ -44,18 +42,8 @@ export function onWindowFocus () {
   saveSettingsLocally()
 }
 
-function onWindowBlur () {
-  if (abortEverything) {
```

Comment:
The `if (abortEverything)` is problematic because it's one of the global variables that record application state. And it's only set to true if the application ends up in an error state (usually during load). So the logic is that if the application is an error state, certain actions in the UI no longer need to be performed. But if we look at it closely, it's this: If the application is an error state, don't hide any menus.

We don't need to do that; `hideAllMenus` checks if any menus are visible, and if they are, hides them. If the application is an error state, then no menus are visible, and therefore, nothing happens.

So while moving the check to `menus.js` I decided it would simplify things not to also import the `abortEverything` global variable.